tasks/11_find-longest-word/find-longest-word.spec.js

```diff
@@ -1,27 +1,27 @@
-// import { findLongestWord } from './find-longest-word';
+import {findLongestWord} from './find-longest-word';
 
-xdescribe('find-longest-word', () => {
-    test('findLongestWord("The quick brown fox jumped over the lazy dog") should return a number', () => {
-        expect(typeof findLongestWord("The quick brown fox jumped over the lazy dog")).toBe('number');
```

Comment:
check if you turn on formatting on save :) you don't need to change something except the first lines in tests files